lib/lib_install.php

```diff
@@ -75,6 +75,103 @@ function checkRequirements(string $dbType = ''): array {
 	);
 }
 
+function getProcessUsername(): string {
+	if (function_exists('posix_getpwuid') && function_exists('posix_geteuid')) {
+		$processUser = posix_getpwuid(posix_geteuid()) ?: [];
+		if (!empty($processUser['name'])) {
+			return $processUser['name'];
+		}
+	}
+
+	if (function_exists('exec')) {
+		exec('whoami', $output);
+		if (!empty($output[0])) {
+			return $output[0];
+		}
+	}
+
+	return _t('install.check.unknown_process_username');
+}
+
+/** @return array<string> */
+function getOwnerOfFile(String $file): array {
```

Comment:
```suggestion
function getOwnerOfFile(string $file): array {
```

attention to the capitalization